package by.test.app.exception;

public class DbNotAvailableException extends RuntimeException {

    public DbNotAvailableException() {
        super();
    }

    public DbNotAvailableException(Throwable cause) {
        super(cause);
    }

}
